LICENSE-binary

```diff
@@ -422,6 +422,11 @@ Python Software Foundation License
 python/pyspark/loose_version.py
 
 
+BSD 0-Clause
```

Comment:
Do we need to update the binary LICENSE file?

Reply:
Oh, right.

Reply:
`licenses/LICENSE-xz.txt` is added.